packages/shared/components/popups/AuthenticationRequiredPopup.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="typescript">
+    import { Button, TextInput, Text, TextType, HTMLButtonType } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import { closePopup } from '@auxiliary/popup'
+    import { handleError } from '@core/error/handlers'
+    import type { Auth } from '@iota/wallet'
+
+    export let onSubmit: (auth: Auth) => unknown = () => {}
+
+    let username: string
+    let password: string
+    let jwt: string
+
+    let usernameError: string
+    let passwordError: string
+    let jwtError: string
+
+    $: disabled = !username && !password && !jwt
```

Comment:
This works differently. You either have a username + password combination OR a JWT. At the moment it enables the button if any character is typed in.

Reply:
We need more input from design team about this and them we can implement an optimal layout

Reply:
You can also have a combination of all three.